docs/compatibility.md

```diff
@@ -180,7 +180,7 @@ date. Typically, one that overflowed.
 
 ### CSV Floating Point
 
-Parsing floating-point values has the same limitations as [casting from string to float](#String-to-Float).
+Parsing floating-point values has the same limitations as [casting from string to float](#string-to-float).
```

Comment:
Is this change pending for `main`?  Otherwise we'll end up fixing this but the docs on `main` will still be broken.  Changes should propagate from branch-XX.YY -> main -> gh-pages.